tests/e2e-http-server/http-custom-certs/50-verify-version.yaml

```diff
@@ -27,7 +27,7 @@ data:
     set -o errexit
 
     # Version can change only validate a portion of it
-    curl --insecure --fail --verbose --key /certs/tls.key --cert /certs/tls.crt https://v-http-custom-certs-main-0.v-http-custom-certs:8443/version | tee /tmp/curl.out
+    curl --insecure --fail --verbose --key /certs/tls.key --cert /certs/tls.crt https://v-http-custom-certs-main-0.v-http-custom-certs:8443/info/version | tee /tmp/curl.out
```

Comment:
Why did you change the url.

Reply:
The server team changed around the API name. I was just updating the test to reflect this. I will eventually get this testsuite (e2e-http-server) into our CI.